<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ccs::main::AddCameraDialog</class>
 <widget class="QDialog" name="ccs::main::AddCameraDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Camera</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Camera backen&amp;d</string>
     </property>
     <property name="buddy">
      <cstring>pluginComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="pluginComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QStackedWidget" name="pluginStackedWidget"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Camera &amp;name</string>
     </property>
     <property name="buddy">
      <cstring>cameraNameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="cameraNameLineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cameraNameLineEdit</tabstop>
  <tabstop>pluginComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ccs::main::AddCameraDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ccs::main::AddCameraDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
